apps/src/templates/RegionalPartnerSearch.jsx

```diff
@@ -321,83 +321,86 @@ class RegionalPartnerSearch extends Component {
                 )}
             </div>
 
-            {appState !== WorkshopApplicationStates.now_closed && (
-              <div>
-                <h3>Workshop information (hosted by {partnerInfo.name}):</h3>
-                {workshopCollections.map((collection, collectionIndex) => {
-                  if (collection.workshops.length === 0) {
-                    // If no current workshops for a course
-                    if (
-                      partnerInfo.pl_programs_offered.includes(collection.key)
-                    ) {
-                      // If a program is offered but a workshop hasn't been scheduled yet
-                      return (
-                        <WorkshopCard
-                          key={collectionIndex}
-                          style={workshopCollectionStyle}
-                          content={
-                            <>
-                              <h4>
-                                {collection.name} Workshop details are coming
-                                soon!
-                              </h4>
-                              <div>
-                                The Regional Partner is hard at work locking
-                                down the details of the workshops for this
-                                program. You can still apply and the Regional
-                                Partner will inform you when the workshop
-                                details are available.
-                              </div>
-                            </>
-                          }
-                        />
-                      );
-                    } else {
-                      // If a program is not offered
+            {appState !== WorkshopApplicationStates.now_closed &&
+              partnerInfo.pl_programs_offered && (
+                <div>
+                  <h3>Workshop information (hosted by {partnerInfo.name}):</h3>
+                  {workshopCollections.map((collection, collectionIndex) => {
+                    if (collection.workshops.length === 0) {
+                      // If no current workshops for a course
+                      if (
+                        partnerInfo.pl_programs_offered.includes(collection.key)
+                      ) {
```

Comment:
This `.includes(...)` was where the error was previously being thrown since `pl_programs_offered` was null

Reply:
Thanks for pointing this out

Reply:
Ahh, I see –– you added that conditional up top

Reply:
It should not run if it's null, it was previously throwing an error calling "includes" on a null object. What is optional chaining?

Reply:
Okay sorry –– should've waited until I fully understood the logic. I blame the World Cup!

Reply:
No worries at all :)